src/components/Layout/Layout.tsx

```diff
@@ -1,11 +1,36 @@
+import Head from "next/head";
 import { NavBar } from "@/components/Layout/NavBar";
 import { Footer } from "@/components/Layout/Footer";
 import { CommonProps } from "@/types";
 
-export const Layout: React.FC<CommonProps> = (props) => (
-  <>
-    <NavBar />
-    <main {...props}>{props.children}</main>
-    <Footer />
-  </>
-);
+const websiteTitle = "La Suite numérique";
+
+export const Layout: React.FC<
+  CommonProps & {
+    /**
+     * Text to use as page <title>. It should be represent what the current page is about.
+     * The website name is appended to the title automatically.
+     */
+    title: string;
+  }
+> = ({ title, ...props }) => {
+  const isHomepage = title === websiteTitle;
+  const pageTitle = isHomepage ? title : `${title} - ${websiteTitle}`;
+  return (
+    <>
+      <Head>
+        <title>{pageTitle}</title>
+        <meta
+          key="ogtitle"
+          property="og:title"
+          content={
+            isHomepage ? "Travailler avec La Suite numérique" : pageTitle
+          }
+        />
+      </Head>
```

Comment:
nit: to save checking for string equality, we could allow title to be optional, and guess that is title is undefined, thus it's the homepage.

Reply:
Then, we avoid having two hard-coded strings l6 in this file and under `pages/index`, that can be desynchronized.

Reply:
also, we could simplify use the constant `websiteTitle` in `pages/index` to avoid duplicating a literal

Reply:
> nit: to save checking for string equality, we could allow title to be optional, and guess that is title is undefined, thus it's the homepage.

I know what you mean but making the title required is actually something I want.

By marking the title optional, the component API literally tells us title is not required. But it is: we don't want any page without a specific title. The homepage is _the_ special case because it is the website title.

But I can totally use a constant for the common title strings to avoid potential issues, good point :)

Reply:
go for it

Reply:
I didn't bother making the specific homepage `og:title` a constant out the component since it's really specific, it's kinda like content in any other page.